controller/contact.js

```diff
@@ -26,21 +41,44 @@ const getById = async (req, res, next) => {
     console.log(error);
     res
       .status(404)
-      .json(genereteJSON("error", 404, "error message", error.message));
-    next(error);
+      .json(
+        genereteJSON(
+          "error",
+          404,
+          "error message",
+          error.message || "Something went wrong",
+        ),
+      );
   }
 };
 
 const create = async (req, res, next) => {
   const { name, email, phone } = req.body;
 
+  const bodyData = { name, email, phone };
+
   try {
+    // I use synchronical validation
+    const validationByJoi = schema.validate(bodyData);
+    // This is example which explains condition in next line.
+    // console.log("Moj blad ", Boolean(validationByJoi.error));
+    //  We have two cases:
+    // In first we have error. Why ? Boolean(validationByJoi) returns true because validationByJoi has error key which value is string.
+    // In second we haven't error. Why? Boolean(validationByJoi) returns undefind. Key error does't exist.
+    if (validationByJoi.error) {
+      return res
+        .status(401)
+        .json(
+          genereteJSON(
+            "error",
+            401,
+            "error message",
+            "Unsuccessful validation",
```

Comment:
Jeśli chodzi o wiadomość przekazywaną do frontendu to po validacji za pomącą Joi warto zwrócić to, co zwraca Joi. Bo może jakiegoś klucza brakuje albo nie spełnia wymagań.

Reply:
Jasne, czyli pogrzebać w Joi'owym obiekcie błędu i coś z niego wyświetlić